src/components/pages/document.yml

```diff
@@ -38,9 +41,18 @@ document:
   saveChanges:
     en: Save changes
     de: Änderungen speichern
+  editActionTooltip:
+    en: Open edit mode
+    de: Bearbeitungs-Modus öffnen
   comments:
     en: Comments
     de: Komentare
+  commentsActionTooltip:
+    en: Open comments mode
+    de: Komentar-Modus öffnen
   history:
     en: Version history
     de: Versionshistorie
+  historyActionTooltip:
+    en: Open version history mode
+    de: Versionshistorie-Modus öffnen
```

Comment:
I have a few suggestion regarding the tootips here:

* I find that "Open xyz mode" sounds a bit technical, also when you go into one of the "modes", the focus header titel is just simply "Comments" or "Version history", so maybe we could change the tooltips accordingly to "Show comments"/"Show version history"/"Edit document"
* As an anonymous user, I get the "Log in in order to xyz" tooltip text for all disabled icons except for the favorite icon (which still just says "Add to favorites"). For even more consistency we could also change this one to "Log in in in order to add the document to your favorites" or similar?

Reply:
Both good points, I've followed the suggestions for both.
Please double check 🙏

Reply:
:+1: